mirdata/download_utils.py

```diff
@@ -8,6 +8,7 @@
 import tarfile
 import urllib
 import zipfile
+import chardet
```

Comment:
is this already a dependency of one of the other libraries? We don't explicitly specify it in setup.py

Reply:
it's a dependency of an existing library. should we put it in setup.py?

Reply:
I think so, yes, just in case one of those libraries someday remove this dependency!